src/Pages/nav.js

```diff
@@ -75,25 +102,48 @@ function NavbarPage() {
                   </MDBDropdownMenu>
                 </MDBDropdown>
               </MDBNavItem>
-              <MDBNavItem>
-                <MDBBtn
-                  color="primary"
-                  className="rounded-pill"
-                  size="sm"
-                  onClick={goToSignUpPage}
-                >
-                  Sign Up
-                </MDBBtn>
-                <MDBBtn
-                  outline
-                  color="primary"
-                  className="rounded-pill"
-                  size="sm"
-                  onClick={goToLoginPage}
-                >
-                  Login
-                </MDBBtn>
-              </MDBNavItem>
+              {user.isSigned && (
+                <>
+                  <MDBNavItem>
+                    <MDBNavLink to="/purchase">
+                      <MDBIcon icon="shopping-cart" />
+                    </MDBNavLink>
+                  </MDBNavItem>
+                  <MDBNavItem>
+                    <MDBNavLink to="/profile">
+                      <MDBIcon icon="user-circle" />
+                    </MDBNavLink>
+                  </MDBNavItem>
+                  <MDBNavItem>
+                    <button
+                      className="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 mx-2 rounded-full"
+                      onClick={onLogout}
+                    >
+                      Logout
+                    </button>
+                  </MDBNavItem>
+                </>
+              )}
+              {!user.isSigned && (
+                <>
+                  <MDBNavItem>
+                    <button
+                      className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 mx-2 border-2 border-transparent rounded-full"
+                      onClick={goToSignUpPage}
+                    >
+                      Sign Up
+                    </button>
+                  </MDBNavItem>
+                  <MDBNavItem>
+                    <button
+                      className="bg-gray-100 hover:bg-blue-500 text-blue-700 font-semibold hover:text-white py-2 px-4 border-2 border-blue-500 rounded-full"
+                      onClick={goToLoginPage}
+                    >
+                      Login
+                    </button>
+                  </MDBNavItem>
+                </>
+              )}
```

Comment:
I think this way is easier to read and I saw it way more is react that ternary

Reply:
but we did something like this in another place using ternary operator 
and I think the other way is better cuz now you are going to do two conditions instead of only one